torchvision/prototype/transforms/_container.py

```diff
@@ -33,9 +34,21 @@ def extra_repr(self) -> str:
 
 
 class RandomChoice(Transform):
-    def __init__(self, transforms: Sequence[Callable], probabilities: Optional[List[float]] = None) -> None:
+    def __init__(
+        self,
+        transforms: Sequence[Callable],
+        probabilities: Optional[List[float]] = None,
+        p: Optional[List[float]] = None,
```

Comment:
Question: Why not send extra things on `kwargs`? This way we expose/advertise on the documentation. 

Also, why do we change to `probabilities`? `p` is quite common in PyTorch.

Reply:
@pmeier I think it was you who introduced `probabilities` to `RandomChoice`.

IMO, `probabilities` emphasizes that it is a list instead of single value like `p`.

> Why not send extra things on kwargs?

We can, but this will look untypical in the code. 

> This way we expose/advertise on the documentation.

I do not understand what you mean by that.

Reply:
> I do not understand what you mean by that.

I mean that the addition of the `p` will get it exposed and advertised in the documentation as a field.

I'm OK with what you do, feel free to keep as you have.